### ISSUE TITLE
False positive: function-calc-no-invalid

### ISSUE THREAD

**Issue by hornta** [stylelint/stylelint#4741]
> Clearly describe the bug

There are false positives for when using less variables like in my code example.

> Which rule, if any, is the bug related to?

function-calc-no-invalid

> What code is needed to reproduce the bug?

e.g.

```css
a {
  font-size: calc(100vh - (@gaming-responsibility-height + @casino-navbar-h + @header-height-desktop + @casino-titlebar-h));
}
```

> What stylelint configuration is needed to reproduce the bug?

e.g.

```json
{
  "rules": {
    "function-calc-no-invalid": true
  }
}
```

> Which version of stylelint are you using?

13.3.3

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

CLI with `stylelint "**/*.less"`

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

It's related to less syntax.

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?

The following warnings were flagged:

```shell
Expected space after "-" operator (function-calc-no-invalid)
Expected space before "-" operator (function-calc-no-invalid)
Expected space after "-" operator (function-calc-no-invalid)
Expected space before "-" operator (function-calc-no-invalid)
```

**Comment by jeddy3** [comment]
@hornta Thanks for the report and for using the template.

I can't reproduce this [using the demo](https://stylelint.io/demo#N4Igxg9gJgpiBcICGACYAdAdilAzCmALgLQDOAlgF4zwphIA2YAFAIwAM7AbgBYrEpmAAQDmSALblMI4gCcYpAA4EKAI3INyhAJ7EeMciJ6EUAahRD6FTBGKYkXVUll6zF-Ulgv9h48VikANaEEIpulkjWtoRaDDBO3gCUiQDcWAC+WCAANCCk2kRIAB4IIHGkpDngBLiGpRjYKOggsgCu5c20DTg4zbitmGAxBMT0THa2UlyM5FCdKIRtMFg4mZjpIOlAA).

Are you sure you're using `13.3.0` as this bug was fixed [in `13.0.0`](https://stylelint.io/changelog#1330)?

You can use `stylelint --version` to check. It's worth double-checking you don't have any globally installed versions of the stylelint either.

I'll close for now, but feel free to reopen with a reproducible repo.

**Comment by hornta** [comment]
I had the incorrect vscode extension of stylelint installed.
I had the lower version installed but the correct one is the above version:
![image](https://user-images.githubusercontent.com/4553604/81263579-333a8000-9040-11ea-873e-cbbf82fc353a.png)
